spec.html

```diff
@@ -39843,6 +39857,82 @@ <h1>Array.from ( _items_ [ , _mapper_ [ , _thisArg_ ] ] )</h1>
         </emu-note>
       </emu-clause>
 
+      <emu-clause id="sec-array.fromAsync">
+        <h1>Array.fromAsync ( _asyncItems_ [ , _mapper_ [ , _thisArg_ ] ] )</h1>
+
+        <p>This async method performs the following steps when called:</p>
```

Comment:
```suggestion
        <h1>Array.fromAsync ( _asyncItems_ [ , _mapper_ [ , _thisArg_ ] ] )</h1>
        <p>This function performs the following steps when called:</p>
```

(AFAICT `Array.from` ought to say "function" instead of "method" too)

Reply:
@linusg: It probably should be "This async function" rather than "This function", right?
This algorithm needs to use the machinery for "built-in async functions" added in #2942.

Reply:
No, the user-exposed method isn’t an `async function` in that sense.

Reply:
I have made an (editorial?) change that changes both Array.from’s and Array.fromAsync’s definitions from “method”s to “function”s.

Reply:
Oh, I didn't realize this depends on #2942. Not really a fan of that kind of abstraction, various promise-returning built-in functions already exist without an additional level of indirection.

Either way, in that case i guess it should say "This async function performs" to use the right definition of `[[Call]]`.

Reply:
Re "This method" vs "This function", see Issue #2576. `Array.from` is mentioned a couple times.